ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -280,6 +283,90 @@ if tflite2_support_is_available
     install: true,
     install_dir: nnstreamer_libdir
   )
+
+
+  if get_option('tflite2-custom-support').enabled() or get_option('tflite2-custom-support').auto()
+    ## Create libtensorflow2-lite-custom.so support.
+
+    ## @todo This assumes the same tflite version with the above.
+    ##       If the user has downloaded libtensorflow-lite manually, it may incur inconsistency.
+    ## @todo This does not use pkgconfig or find_library/meson-dep info
+    ##       because it does not allow querying libpath and static/shared of a dependency object.
+    ## @todo If we are allowed to use meson > 0.53, use "FS" module!
+
+    filepaths = [
+        join_paths(meson.current_source_dir(), 'libtensorflow2-lite.so',),
+        join_paths(meson.current_build_dir(), 'libtensorflow2-lite.so',),
+        join_paths(nnstreamer_libdir, 'libtensorflow2-lite.so'),
+        '/usr/lib/libtensorflow2-lite.so',
+        '/usr/lib64/libtensorflow2-lite.so',
+        './libtensorflow2-lite.so',
+    ]
+    filepath = ''
+    foreach file : filepaths
+      if run_command('bash', '-c', '[ -f "' + file + '" ]').returncode() == 0
+        message('Found: ' + file)
+        filepath = file
+        configure_file (
+          input: file,
+          output: 'libtensorflow2-lite-custom.so',
+          copy: true
+        )
+        break
+      endif
+    endforeach
+
+    if filepath == ''
+      ## No shared tfl lib found. Get static lib and convert it
+
+      filepaths = [join_paths(nnstreamer_libdir, 'libtensorflow2-lite.a'),
+          '/usr/lib/libtensorflow2-lite.a',
+          '/usr/lib64/libtensorflow2-lite.a',
+          './libtensorflow2-lite.a',
+          join_paths(meson.current_build_dir(), 'libtensorflow2-lite.a',),
+          join_paths(meson.current_source_dir(), 'libtensorflow2-lite.a',)]
```

Comment:
LGTM👍 Minor concern: the order of L297~L303 and L322~L327 should be same?

Reply:
No. they were ordered differently intentionally.

.a is supposed to be provided by our tf-lite-dev package (the first three cases) primarily.

.so is supposed to be provided by the builder (we don't release tf-lite so in our packages) as an ad-hoc method; thus, they are supposed to be at the current "source" or "bin" directory.